src/angular/projects/spark-core-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.ts

```diff
@@ -91,6 +124,8 @@ export class SparkDropdownComponent {
   @Input()
   title: string;
   @Input()
+  selector: string;
```

Comment:
I see there's an Input selector and also a value for dropdownType 'mastheadSelector', do we need both?

Reply:
@RCopeland the selector input string is for the masthead selector title. So we need it.